doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -984,7 +994,7 @@ Binding types of arguments to an interpretation scope
 
    :cmd:`About`
      The command to show the scopes bound to the arguments of a
-     function is described in Section 2.
+     function is described in _extensionsofgallina.
```

Comment:
According to the last version of the manual before the migration https://coq.inria.fr/distrib/V8.7.2/refman/syntax-extensions.html#scopes, the link was just to the definition of the command `About` (https://coq.inria.fr/distrib/V8.7.2/refman/vernacular.html#About). So you can probably remove this second link and rephrase the whole sentence.

Reply:
This sentence actually seems out of place, and there's another `seealso` block pointing to that command in the subsection above. So I'll remove it.